teachapp/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,55 @@
+class UsersController < ApplicationController
+ 
+  def index
+    @users = User.all
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      # render success in Jbuilder
```

Comment:
use `unless` to do something in the case it fails, otherwise just continue and render :create as per normal